packages/react-native/React/Base/RCTBundleURLProvider.h

```diff
@@ -118,20 +119,38 @@ RCT_EXTERN void RCTBundleURLProviderAllowPackagerServerAccess(BOOL allowed);
  * - enableDev: Whether to keep or remove `__DEV__` blocks from the bundle.
  * - enableMinification: Enables or disables minification. Usually production bundles are minified and development
  *     bundles are not.
+ * - inlineSourceMap: Determines whether the source map is inline in the JS bundle. This is useful for debugging with Safari /JavascriptCore.
  * - modulesOnly: When true, will only send module definitions without polyfills and without the require-runtime.
  * - runModule: When true, will run the main module after defining all modules. This is used in the main bundle but not
  *     in split bundles.
  */
 + (NSURL *)jsBundleURLForBundleRoot:(NSString *)bundleRoot
                        packagerHost:(NSString *)packagerHost
                           enableDev:(BOOL)enableDev
-                 enableMinification:(BOOL)enableMinification;
+                 enableMinification:(BOOL)enableMinification
+__deprecated_msg("Use version with inlineSourceMap parameter instead");
```

Comment:
nit: i will probably use the full signature:
```suggestion
__deprecated_msg("Use `jsBundleURLForBundleRoot:packagerHost:enableDev:enableMinification:inlineSourceMap:` instead")
```
Swift has also `renamed`, which will suggest an autofix. :( Too bad we can't use it.